_liveDeploys/liveDeploy.md

```diff
@@ -74,6 +74,16 @@ list:
     bsc_ver: 0.1
     comments:
     node: UK
+-
+    name: IUPHAR/BPS Guide to PHARMACOLOGY
+    highlight: over 9,800 ligands
+    example_URL: https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=8299
+    resource_URL: https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=8299
```

Comment:
you are changing the meaning/use of resource_URL. It normally points to the home page of the resource. Is this deliberate?

Reply:
Probably best if we are consistent with other descriptions. I guess we may need to think about the page layout at some point since it would be nice to be able to click to the page with the markup on as well as the resource home page.

Reply:
Now fixed